src/App.js

```diff
@@ -33,7 +33,7 @@ function App({ settingsManager }) {
 
             <Switch>
               <Route path="/settings" exact>
-                <SettingsPage settingsManager={settingsManager} />
+                <SettingsPage settingsManager={settingsManager} hasRecording={activeStep === 3}/>
```

Comment:
I do not think that the SettingsPage should know about some active step number of another page. This way the state is all over the place and I would prefer to do in another way.